tests/run_blivet.yml

```diff
@@ -1,7 +1,13 @@
 ---
+- include_vars:
+    dir: roles/storage/defaults
+    extensions: ['yml']
```

Comment:
What's this for?  I guess this is to have access to the default role variables in the test, before the role is included?  Can you do `include_vars:  name: roles/storage/defaults/main.yml`?

Reply:
That is correct and sure -- I can explicitly include the main.yml. That's what I wanted to do anyway.